Assets/HoloToolkit/Input/Scripts/InputSources/IInputSource.cs

```diff
@@ -119,6 +119,11 @@ public interface IInputSource
         event EventHandler<NavigationEventArgs> NavigationCanceled;
 
         /// <summary>
+        /// Event triggered when a speech phrase is recognized.
+        /// </summary>
+        event EventHandler<PhraseRecognizedEventArgs> PhraseRecognized;
```

Comment:
@maxouellet I just noticed that changing to SpeechKeywordRecognizedEventArgs avoids having to use explicit namespaces because of name clashing. I would then change the IInputSource event to SpeechKeywordRecognized. What do you think?